recipes/mold/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.45.0"
+
+class MoldConan(ConanFile):
+    name = "mold"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rui314/mold/"
+    license = "Zlib"
```

Comment:
```suggestion
    license = "AGPL-3.0"
```
https://github.com/rui314/mold/blob/v1.3.1/LICENSE
https://spdx.org/licenses/AGPL-3.0.html